pkg/agent/openflow/network_policy.go

```diff
@@ -945,12 +982,18 @@ func (c *client) UninstallPolicyRuleFlows(ruleID uint32) error {
 	ctxChanges := conj.calculateChangesForRuleDeletion()
 	// Send the changed OpenFlow entries to the OVS bridge and update the conjMatchFlowContext.
 	if err := c.applyConjunctiveMatchFlows(ctxChanges); err != nil {
-		return err
+		return nil, err
 	}
 
-	// Remove policyRuleConjunction from client's policyCache.
-	c.policyCache.Delete(ruleID)
-	return nil
+	c.policyCache.Delete(conj)
+	for _, p := range ofPrioritiesToUninstallFlows {
+		conjsStalePriority, _ := c.policyCache.ByIndex(priorityIndex, p)
+		if len(conjsStalePriority) == 0 {
+			klog.V(2).Infof("ofPriority %v is now stale", p)
+			staleOFPriorities = append(staleOFPriorities, p)
+		}
+	}
```

Comment:
what does this mean? one OFPriority can be shared by multiple rules?

Reply:
Yes it is entirely possible.